src/browser/BrowserCodeReader.ts

```diff
@@ -112,6 +112,22 @@ export class BrowserCodeReader {
     return this._hints;
   }
 
+  protected _timeBetweenDecodingTries: number = 0;
```

Comment:
What you think about we calling this `_timeBetweenDecodeAttempts`?

Reply:
No concerns.